lib/src/widgets/flutter_deck_footer.dart

```diff
@@ -66,16 +75,19 @@ class FlutterDeckFooter extends StatelessWidget {
         mainAxisAlignment: MainAxisAlignment.spaceBetween,
         crossAxisAlignment: CrossAxisAlignment.end,
         children: [
-          if (showSocialHandle)
+          if (widget != null)
```

Comment:
To avoid the `widget!`, you can do the fancy pattern matching trick there:
```suggestion
          if (widget case final widget?)
```

Reply:
I find the syntax with the question mark pretty confusing and went for the more explicit one: `if (widget case final Widget widget)`. Is that one fine as well?